docs/glossary.md

```diff
@@ -11,74 +6,116 @@ sidebar_position: 27
 
 # Glossary
 
+List of frequently used terms and acronyms.
+
 <a id="term-ABI"></a>
 
 ## ABI
 
-**A**pplication **B**inary **I**nterface. ABI is a document that comprehensively defines the binary system interface between applications and the operating system on which they run. [Learn More at Wikipedia](https://en.wikipedia.org/wiki/Application_binary_interface) or [pypackaging-native](https://pypackaging-native.github.io/background/binary_interface).
+Application Binary Interface. ABI is a document that comprehensively defines the binary system interface between applications and the operating system on which they run.
+
+[Learn more](https://en.wikipedia.org/wiki/Application_binary_interface).
+
+<a id="term-CDN"></a>
+
+## CDN
+
+Content Delivery Network. CDNs are geographically distributed networks of servers that mirror the contents of a primary source. Having multiple servers offering the same content increases performance (reduced latency, higher download speeds) and availability.
+
+[Learn more](https://en.wikipedia.org/wiki/Content_delivery_network).
 
 <a id="term-CDT"></a>
 
 ## CDT
 
-**C**ore **D**ependency **T**ree. Core Dependency Tree packages take care of the dependencies which are so close to the system that they are not packaged with conda-forge. A CDT package consists of repackaged CentOS binaries from the appropriate version, either 6 or 7 depending on user choice and platform. [Learn more](maintainer/knowledge_base.md#cdt-packages).
+Core Dependency Tree. Core Dependency Tree packages take care of the dependencies that are so close to the system that they are not packaged with conda-forge. A CDT package consists of repackaged CentOS binaries from the appropriate version, either 6 or 7 depending on user choice and platform.
```

Comment:
This part will get out of date soon. I think it's better to have this last sentence present an example.

> For example, in conda-forge, we have used repackaged CentOS 6 or 7 for some time.